partner_lastname_uppercase/data/actions.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_partner_uppercase_lastname_action" model="ir.actions.server">
```

Comment:
shouldn’t `uppercase` and `lastname` follow the same order as in the module name?